contracts/orbital-core/src/admin_logic.rs

```diff
@@ -0,0 +1,51 @@
+pub(crate) mod admin {
+    use cosmwasm_std::{ensure, Addr, BlockInfo, MessageInfo, Response};
+    use cw_ownable::{assert_owner, update_ownership, Action};
+
+    use crate::{
+        contract::{ExecuteDeps, OrbitalResult},
+        error::ContractError,
+        orbital_domain::UncheckedOrbitalDomainConfig,
+        state::ORBITAL_DOMAINS,
+    };
+
+    pub fn transfer_admin(
+        deps: ExecuteDeps,
+        block: &BlockInfo,
+        sender: &Addr,
+        action: Action,
+    ) -> OrbitalResult {
+        let resp = update_ownership(deps.into_empty(), block, sender, action)
+            .map_err(ContractError::Ownership)?;
```

Comment:
this map_err is not needed right? You already have the Ownership errors imported there

Reply:
I'm afraid a workaround at some place is needed  due to the neutron specific execute deps and `NeutronResult`

Reply:
right that's why honestly I'm not a big fan of using NeutronResult (over Result<Response<NeutronMsg>, ContractError>) this way you just map the neutron errors, but I guess it's fine